vars/runTestFunctionalV2.groovy

```diff
@@ -53,6 +55,7 @@ void call(Map config = [:]) {
                        'PRAGMA_SUFFIX="' + config['pragma_suffix'] + '" ' +
                        'NODE_COUNT="' + config['node_count'] + '" ' +
                        'OPERATIONS_EMAIL="' + env.OPERATIONS_EMAIL + '" ' +
+                       "WITH_VALGRIND=${stage_info.get('with_valgrind', '')} " +
```

Comment:
I don't see where this is consumed in https://github.com/daos-stack/daos/blob/0e62a8ade4bc4e7af080f867b6191c8951ff428c/src/tests/ftest/scripts/main.sh.  Rather, I see it's value being overridden:
```
if [[ "${TEST_TAG_ARG}" =~ memcheck ]]; then
  export WITH_VALGRIND="memcheck"
fi
```
by using the test tag value as a proxy for just using the passed in value of this variable.  It seems that using the value of this variable, as passed in is more accurate than overriding it based on test tag values.

Reply:
That `WITH_VALGRIND` setting is from an earlier experiment, so I removed it. Good catch. 👍